app/assets/javascripts/proxy_status/logs.js

```diff
@@ -0,0 +1,77 @@
+function filterLogsReset() {
+  var table = $('#table-proxy-status-logs').dataTable();
+  for (var i = 0; i <= 2; i++) {
+    table.fnFilter('', i);
+  }
+}
+
+function filterLogsByLevel(filter) {
+  filterLogsReset();
+  var table = $('#table-proxy-status-logs').dataTable();
+  table.fnFilter(filter, 1, true, false);
+}
+
+function filterLogsByMessage(expression) {
+  filterLogsReset();
+  changeFilterSelection(1);
+  var table = $('#table-proxy-status-logs').dataTable();
+  table.fnFilter('ERROR|FATAL', 1, true, false);
+  table.fnFilter(expression, 2, true, false);
+}
+
+function changeFilterSelection(index) {
+  var filter = $('#logs-filter');
+  filter[0].options[index].selected = true
+  filter.select2();
+  filterLogsByLevel(filter.val());
+}
+
+function activateLogsDataTable() {
+  $('#table-proxy-status-logs').dataTable({
+    "sDom": "<'row'<'col-md-6'f>r>t<'row'<'col-md-6'i><'col-md-6'p>>",
+    "sPaginationType": "bootstrap",
+    "bSort": false,
+    "aoColumnDefs": [{
+      "mRender": function ( data, type, row ) {
+        var formatted;
+        try {
+          formatted = new Date(data * 1000).toLocaleString();
+        } catch (e) {
+          formatted = new Date(data * 1000).toGMTString();
+        }
+        return formatted;
+      }, "aTargets": [0]}]
+    });
+  var filter = $('#logs-filter');
+  filter.select2();
+  filter.on('change', function() { filterLogsByLevel(filter.val()) });
+
+  $('#backtrace-modal').on('show.bs.modal', function (event) {
+    var link = $(event.relatedTarget);
+    var modal = $(this);
+    var datetime = new Date(link.data('time') * 1000);
+    try {
+      modal.find('#modal-bt-timegmt').text(datetime.toGMTString());
```

Comment:
`toGMTString` is deprecated. Please use `toUTCString` instead. [[1]](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toGMTString)